### DISCUSSION TITLE
Athom 16A EU V2

### DISCUSSION THREAD

**Discussion by MachaVaclav** [arendst/Tasmota#17070] [Q&A]
Hi,
I bought the Athom 16A EU V2 Power Monitoring Plug (PG01V2-EU16A-TAS).
I turn off WiFi without WiFi it didn't work for about 30 minutes then it turned on.
I had to upload Tasmota 9.4.0 by Theo Arends which I think is quite an old FW
Regards

**Comment by Jason2866** [comment]
What is your question?

**Reply by MachaVaclav** [reply]
I turn off the WiFi without WiFi it didn't work for about 30 minutes and then it turned on.
I had to upload Tasmota 9.4.0 by Theo Arends to get it to work, which I think is quite an old FW

**Comment by MachaVaclav** [comment]
Latest 12.2.0.4

**Reply by MachaVaclav** [reply]
I uploaded the latest fw 12.2.0.4 Athom 16A EU V2. I turn off WiFi at night. About 30 minutes and then it turned on. I had to upload Tasmota 9.4.0 by Theo Arends to make it work I think it's old FW
